internal/core/src/index/Utils.cpp

```diff
@@ -59,6 +59,18 @@ DISK_LIST() {
     return ret;
 }
 
+std::vector<IndexType>
+RAW_DATA_LIST() {
```

Comment:
It is better to have knowhere provide an interface to check if the original vector is included, rather than maintaining the list here.